program-test/Cargo.toml

```diff
@@ -15,6 +15,7 @@ base64 = { workspace = true }
 bincode = { workspace = true }
 chrono-humanize = { workspace = true }
 crossbeam-channel = { workspace = true }
+lazy_static = { workspace = true }
```

Comment:
Seems unused. And if it were used we should switch to https://doc.rust-lang.org/std/cell/struct.OnceCell.html

Reply:
I was using it previously but removed the need for it, so yeah it's now unused.